docs/001-custom-features.md

```diff
@@ -0,0 +1,33 @@
+# Custom features for SGA.
+
+## Grading for CCX coaches.
+
+### Context
+
+Out-of-the-box, SGA only allows the main-course admin users (instructors in the course access role model) to grade assignments.
+
+This image shows the SGA view where course admins users can grade student submissions. As shown, the CCX coach cannot grade student submissions:
+
+![](../docs/images/001-01.png "001-01")
+
+This image shows the same SGA view, but with an admin role, indicating that only admin users of the course can grade student submissions:
+
+![](../docs/images/001-02.png "001-02")
+
+### Feature
+
+This feature allows CCX coaches (ccx_coach in the course access role model) and CCX admin users (instructor in the course access role model) to grade assignments in their CCX courses.
+
+![](../docs/videos/001-02.gif "001-01")
+
+### Testing
```

Comment:
It's neccesary and the testing instructions in the ADR? It is not enough since it is in the PR?

Reply:
This is not an ADR, it's documentation about the custom features. @anfbermudezme